libfreerdp/core/nla.c

```diff
@@ -2315,6 +2312,13 @@ rdpNla* nla_new(freerdp* instance, rdpTransport* transport, rdpSettings* setting
 	}
 
 	return nla;
+
```

Comment:
wouldn't calling `nla_free` be the better option (and check, if that does proper ceanup)?